flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FuelPredPage.js

```diff
@@ -148,9 +148,17 @@ class CDUFuelPredPage {
                 };
                 if (mcdu.altDestination) {
                     altIdentCell = mcdu.altDestination.ident;
-                    altEFOBCell = (mcdu.getAltEFOB(true) * mcdu._conversionWeight).toFixed(1);
-                    altTimeCellColor = "[color]green";
+                    const altEFOBCellVal = (mcdu.getAltEFOB(true) * mcdu._conversionWeight).toFixed(1);
                     altEFOBCellColor = "[color]green";
+                    altEFOBCell = altEFOBCellVal;
+                    if (altEFOBCellVal < 0.8) {
```

Comment:
As per a reference I have it is not exactly below 0.8, it's below whatever the final reserve is (which comes out to 0.8 by default at the moment, but can be different).

Reply:
ALT DEST EFOB value turns amber when less than FINAL value.

Please provide next time your references and also provide a possible solution regarding calculations.

Reply:
done